convert/testdata/Phi-3-mini-128k-instruct.json

```diff
@@ -0,0 +1,223 @@
+{
+  "general.architecture": "phi3",
+  "general.file_type": "1",
+  "phi3.block_count": "32",
+  "phi3.context_length": "131072",
+  "phi3.embedding_length": "3072",
+  "phi3.feed_forward_length": "8192",
+  "phi3.rope.scaling.original_context_length": "4096",
+  "phi3.rope.dimension_count": "96",
+  "phi3.rope.freq_base": "10000",
+  "phi3.rope.scaling.attn_factor": "1.1902381",
+  "phi3.attention.head_count": "32",
+  "phi3.attention.head_count_kv": "32",
+  "phi3.attention.layer_norm_rms_epsilon": "1e-05",
+  "tokenizer.ggml.model": "llama",
+  "tokenizer.ggml.pre": "default",
+  "tokenizer.ggml.add_bos_token": "true",
+  "tokenizer.ggml.add_eos_token": "false",
+  "tokenizer.ggml.bos_token_id": "1",
+  "tokenizer.ggml.eos_token_id": "32000",
+  "tokenizer.ggml.unknown_token_id": "0",
+  "tokenizer.ggml.padding_token_id": "32000",
+  "tokenizer.ggml.scores": "6e37bcde2adc7e350e87c496eddd7a2124329c1dc66c5bf3ad3997253e4f7a62",
+  "tokenizer.ggml.token_type": "b6ecf55ec64ee67d87750bdb8d757a2c58bf78377e9f4219f5689a6c4dea57ce",
+  "tokenizer.ggml.tokens": "d168da3ddd3eee820916945fcb9baf24dd3cde42f606cffa2d19e7c8a8743918",
+  "rope_factors_long.weight": "8a8bfb3c2dc2b86ad3c04af2b50254a9879a0b7605936605613bf569e359e866",
+  "rope_factors_short.weight": "2413d459801123eb2cb7a31b599411d2d4235cfb15b9627fb09ce31b27428c4c",
+  "token_embd.weight": "1f14e80e4f39cb13666b20421f5545634f4f1821e9dc09ca88abc1e8fd2bcbd9",
+  "blk.0.attn_norm.weight": "1339abad65d8e8b93ac6d741e65f997e2954c936836ce588741956f805b1c292",
```

Comment:
Might be a silly question but why is this a checksum?

Reply:
these are the checksum of the tensors. it's the best way I've found to verify the tensors are written properly